migration/migration.go

```diff
@@ -0,0 +1,29 @@
+package migration
+
+import (
+	"context"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/microstorage"
+)
+
+func Migrate(ctx context.Context, src, dest microstorage.Storage) error {
```

Comment:
The order in golang I am used to is usually `dst` before `src`. See https://golang.org/pkg/builtin/#copy for instance.